docs/user-guide/features.rst

```diff
@@ -331,13 +331,320 @@ Modules and their artifacts (RPMs), complicated case 2
    ``recursive_conservative`` takes precedence.
 
 
-Erratum and related RPMs
-^^^^^^^^^^^^^^^^^^^^^^^^
+Erratum and related RPMs/Modules
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Erratum references RPMs and/or Modules.
+| Erratum references RPMs and/or Modules.
+| An Erratum lists RPMs which are suggested to be updated.
+| In case a Module should be updated, an Erratum lists a Module and all its artifacts.
 
 In case of a recursive copy in addition to the copy of Erratum itself, referenced RPMs
-and Modules are copied as well according to the rules and examples explained in previous sections.
+and Modules are copied as well similar to the rules and examples explained in previous sections.
+
+Non-modular Errata and related RPMs, simple case
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+| A non-modular Erratum lists RPMs which are suggested to be updated.
+| Simple case (no RPM dependencies).
+
+::
+
+   erratum-FOO: [foo-1.0.rpm]
+
+   repo A
+     |
+     |----erratum-FOO
+     |----foo-1.0.rpm
+
+
+   repo B
+     |
+     |----foo-0.7.rpm
+
+
+
+| Use case #1 (not recommended): copy erratum itself
+| Flag to use: None
+
+::
+
+    Result of copying erratum-FOO from repo A to repo B:
+
+    repo B
+     |
+     |----erratum-FOO
+     |----foo-0.7.rpm
+
+    Erratum is copied, while its related RPM is not!
+    RPMs which are suggested for update are not in repo B!
+    Copy this way when you know what you are doing and why.
+
+
+| Use case #2: copy erratum and its related RPMs
+| Flag to use: ``recursive`` or ``recursive_conservative``
+
+::
+
+    Result of copying erratum-FOO from repo A to repo B:
+
+    repo B
+     |
+     |----erratum-FOO
+     |----foo-1.0.rpm
+     |----foo-0.7.rpm
+     |----bar-0.7.rpm
+
+    Even though older version of foo is in the repo B,
+    with any recursive flag foo-1.0 is copied anyway
+    because erratum-FOO refers to it.
+
+
+Non-modular Errata and related RPMs, complicated case
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+| A non-modular Erratum lists RPMs which are suggested to be updated.
+| Complicated case (RPM dependencies).
+
+::
+
+   dependencies: foo.rpm -> bar.rpm -> baz.rpm
+   erratum-FOO: [foo-1.0.rpm]
+
+   repo A
+     |
+     |----erratum-FOO
+     |----foo-1.0.rpm
+     |----bar-1.0.rpm
+     |----baz-1.0.rpm
+
+
+   repo B
+     |
+     |----foo-0.7.rpm
+     |----bar-0.7.rpm
+
+
+| Use case #1: copy erratum and related RPMs and RPMs' *latest* dependencies
+| Flag to use: ``recursive``
+
+::
+
+    Result of copying erratum-FOO from repo A to repo B:
+
+    repo B
+     |
+     |----erratum-FOO
+     |----foo-1.0.rpm
+     |----bar-1.0.rpm
+     |----baz-1.0.rpm
+     |----foo-0.7.rpm
+     |----bar-0.7.rpm
+
+| Use case #2: copy erratum and related RPMs and RPMs' *missing* dependencies
+| Flag to use: ``recursive_conservative``
+
+::
+
+    Result of copying module-FOO from repo A to repo B:
+
+    repo B
+     |
+     |----erratum-FOO
+     |----foo-1.0.rpm
```

Comment:
``foo-1.0.rpm`` RPM being copied maybe should quickly re-iterated again in a sentence below this stellar example.

While ``erratum-FOO`` should be copied, it may be assumed that since ``foo-0.7.rpm`` already exists on repo B and would not be copied. 

It may be helpful to re-iterate that ``erratum-Foo`` and ``foo-1.0.rpm`` will always be copied as they are the source of the update. If the update contained RPM updates AND one of their dependencies, BOTH RPMS and the errata would be copied ( ``erratum-FOO: [foo-1.0.rpm, bar-1.0.rpm]``).

That example isn't explicitly required, it would be nice to explicity explain why ``foo-1.0.rpm`` will always get copied.